templates/web-application/package.json

```diff
@@ -18,58 +18,69 @@
         "react-dom": "18.2.0",
-        "react-router-dom": "6.11.1"
+        "react-router-dom": "6.11.2"
     },
     "devDependencies": {
         "@pmmmwh/react-refresh-webpack-plugin": "0.5.10",
-        "@storybook/addon-essentials": "7.0.12",
-        "@storybook/addon-links": "7.0.12",
-        "@storybook/blocks": "7.0.12",
-        "@storybook/react": "7.0.12",
-        "@storybook/react-webpack5": "7.0.12",
+        "@storybook/addon-essentials": "7.0.18",
+        "@storybook/addon-links": "7.0.18",
+        "@storybook/blocks": "7.0.18",
+        "@storybook/react": "7.0.18",
+        "@storybook/react-webpack5": "7.0.18",
         "@storybook/testing-library": "0.1.0",
         "@svgr/webpack": "8.0.1",
-        "@swc/core": "1.3.58",
+        "@swc/core": "1.3.61",
         "@swc/helpers": "0.5.1",
         "@swc/jest": "0.2.26",
         "@testing-library/react": "14.0.0",
-        "@types/jest": "29.5.1",
+        "@types/jest": "29.5.2",
         "@types/lodash-es": "4.17.7",
-        "@types/node": "18.16.10",
-        "@types/react": "18.2.6",
+        "@types/node": "18.16.16",
+        "@types/react": "18.2.7",
         "@types/react-dom": "18.2.4",
-        "@workleap/browserslist-config": "1.2.0",
-        "@workleap/eslint-plugin": "1.6.0",
-        "@workleap/postcss-plugin": "1.2.1",
-        "@workleap/stylelint-config": "1.2.0",
-        "@workleap/typescript-configs": "2.2.0",
-        "browserslist": "4.21.5",
-        "css-loader": "6.7.3",
-        "eslint": "8.40.0",
+        "@workleap/browserslist-config": "1.2.1",
+        "@workleap/eslint-plugin": "1.8.1",
+        "@workleap/postcss-plugin": "1.2.2",
+        "@workleap/stylelint-config": "1.2.1",
+        "@workleap/typescript-configs": "2.3.1",
+        "browserslist": "4.21.7",
+        "css-loader": "6.8.1",
+        "eslint": "8.41.0",
         "html-webpack-plugin": "5.5.1",
         "jest": "29.5.0",
         "jest-environment-jsdom": "29.5.0",
+        "jest-junit": "16.0.0",
         "lodash-es": "4.17.21",
         "msw": "1.2.1",
-        "storybook": "7.0.12",
+        "storybook": "7.0.18",
         "storybook-addon-swc": "1.2.0",
-        "style-loader": "3.3.2",
-        "stylelint": "15.6.1",
+        "style-loader": "3.3.3",
+        "stylelint": "15.6.2",
         "swc-loader": "0.2.3",
-        "terser-webpack-plugin": "5.3.8",
+        "terser-webpack-plugin": "5.3.9",
         "typescript": "5.0.4",
-        "webpack": "5.82.1",
+        "webpack": "5.85.0",
         "webpack-bundle-analyzer": "4.8.0",
         "webpack-cli": "5.1.1",
         "webpack-dev-server": "4.15.0"
     },
     "msw": {
         "workerDirectory": "public"
+    },
+    "jest-junit": {
+        "outputDirectory": "reports",
+        "outputName": "jest-junit.xml",
+        "ancestorSeparator": " › ",
+        "uniqueOutputName": "false",
+        "suiteNameTemplate": "{filepath}",
+        "classNameTemplate": "{classname}",
+        "titleTemplate": "{title}"
```

Comment:
Is it possible to extract this configuration in a standalone configuration file at the root of the template?